deploy_staging.sh

```diff
@@ -1,6 +1,6 @@
 source $(dirname $0)/deploy.sh
```

Comment:
This file is same as `deploy_prod.sh`. Two files can be merged and settings variable can be set by inspecting appropriate environment variable.